apiserver/facades/agent/uniter/uniter.go

```diff
@@ -1599,6 +1616,12 @@ func (u *UniterAPI) SLALevel() (params.StringResult, error) {
 }
 
 // NetworkInfo returns network interfaces/addresses for specified bindings.
+func (u *UniterAPIV6) NetworkInfo(args params.NetworkInfoParams) (params.NetworkInfoResults, error) {
```

Comment:
The semantics are different and that's why we also need the version bump. NetworkInfoParams gains an optional RelationId attribute. We don't want callers expecting V7 behaviour to accidentally call a v6 API. The behaviour between V6 and v7 is indeed identical if the relation id is omitted when calling v7. So that's why v6 NetworkInfo simply delegates to v7 NetworkInfo.